client/components/happychat/style.scss

```diff
@@ -52,6 +52,10 @@
 	right: 12px;
 	font-size: 14px;
 	display: none;		/* disabled */
+	box-sizing: border-box;
+
+	transform: translateZ(0);
```

Comment:
are these to force composition on a gpu layer? might try `will-change` too? https://developer.mozilla.org/en-US/docs/Web/CSS/will-change